test/common/services/security/user.ts

```diff
@@ -49,7 +49,7 @@ export class User {
         `Expected status code of 200, received ${status} ${statusText}: ${util.inspect(data)}`
       );
     }
-    this.log.debug(`created user ${username}`);
+    // this.log.debug(`created user ${username}`);
```

Comment:
any reason to comment out logging here?

Reply:
eliminating too much of logging. Will remove the comment .